zwave_js_server/model/version.py

```diff
@@ -18,4 +20,10 @@ def from_message(cls, msg: dict) -> "VersionInfo":
             driver_version=msg["driverVersion"],
             server_version=msg["serverVersion"],
             home_id=msg["homeId"],
+            # schema versions are sent in the response from schema version 1+
+            # this info not present means the server is at schema version 0
+            # at some point in time (when we stop supporting schema version 0),
+            # we could adjust this code and assume the keys are there.
+            min_schema_version=msg.get("minSchemaVersion", 0),
```

Comment:
I think we should add a test where we send data from a version 0 server, ie not including these new items. We want to check that we're allowing that here.

Reply:
Yes, I did not add any additional tests for added logic yet. I'll see what I can do later today.